UI/scripts/view.js

```diff
@@ -0,0 +1,90 @@
+
+const dateConfig = {
+  year: 'numeric',
+  month: 'numeric',
+  day: 'numeric',
+  hour: 'numeric',
+  minute: 'numeric',
+};
+
+class View {
+  constructor(model, isLoggedUser) {
+    this.photoPosts = model;
+    this.isLoggedUser = isLoggedUser;
+  }
+
+  addPost(post) {
+    if (this.photoPosts.addPhotoPost(post)) {
+      const wrapper = document.querySelector('#wrapper');
```

Comment:
It will be nice to cache it in constructor and it will be ideal to define it outside of view instance.